lib/jwt/taskrouter/util.js

```diff
@@ -53,6 +53,11 @@ function defaultWorkerPolicies(version, workspaceSid, workerSid) {
 function defaultEventBridgePolicies(accountSid, channelId) {
   var url = _.join([EVENT_URL_BASE, accountSid, channelId], '/');
   return [
+    new Policy({
+      url: EVENT_URL_BASE,
+      method: 'GET',
+      allow: true
+    }),
```

Comment:
I'm not seeing this new policy as being required. Is there an issue you're running into?

Reply:
Yes, I'm trying to use https://github.com/twilio/twilio-taskrouter.js. If you don't include the policy then the token returns a `403` because is expecting this permission.